account_invoice_margin/__manifest__.py

```diff
@@ -13,7 +13,7 @@
     "maintainers": ["sergio-teruel"],
     "application": False,
     "installable": True,
-    "depends": ["account"],
+    "depends": ["account", "sale"],
```

Comment:
Why adding this dependency?

Reply:
The test needs this dependency

Reply:
Then there's something incorrect. There's the module `account_invoice_margin_sale` that makes the link.

Reply:
so we should depend on this module?
account_invoice_margin_sale depends on this module

Reply:
No, this module shouldn't depend on sale on tests.

Reply:
So we must modify the test and delete the sale.order